interfaces/cython/cantera/thermo.pyx

```diff
@@ -632,7 +632,30 @@ cdef class ThermoPhase(_SolutionBase):
         stoichAirFuelRatio = - (Of - 2*Cf - 2*Sf - Hf/2.0) / (Oo - 2*Co - 2*So - Ho/2.0)
         Xr = phi * Xf + stoichAirFuelRatio * Xo
         self.TPX = None, None, Xr
+        
+    def get_equivalence_ratio(self):
+        """Considers the oxidation of C to CO2, H to H2O and S to SO2.
+        Other elements are assumed not to participate in oxidation
+        (i.e. N ends up as N2).
+        O2 is considered as the only oxidizer (i.e. NO is considered a "fuel")
```

Comment:
Sorry, and two more here. Can we add an argument to the function definition that allows the user to specify the oxidizer species, with the default being `'O2'`? Would that complicate the logic awfully? Is it worth it to do that?

It would also be nice if this docstring was a little more complete (see the `set_equivalence_ratio` immediately above function for an example).

Reply:
Interesting idea. I could add a dict argument to specify the oxidizer, with the default being `{'O': 2}`. That would allow "non-traditional" oxidizers, like ozone, atomic oxygen, or NO.

Reply:
Now that I think about it more, maybe it's most accurate to have the oxidizer defined as any species that contains oxygen atoms that doesn't contain sulfur, carbon, or hydrogen. Then it would automatically include ozone, atomic oxygen, and NOx